webpack/ForemanInventoryUpload/Components/Dashboard/DashboardActions.js

```diff
@@ -66,16 +71,47 @@ export const setActiveTab = (accountID, tabName) => ({
   },
 });
 
-export const downloadReports = accountID => {
-  const url = inventoryUrl(`${accountID}/uploads/file`);
-  return get({
-    key: INVENTORY_REPORTS_DOWNLOAD,
-    url,
-    toastError: error => error.response.message,
-    handleSuccess: () => {
-      window.location.href = url;
-    },
-  });
+export const downloadReports = accountID => async dispatch => {
+  dispatch({ type: REPORT_DOWNLOAD_REQUEST, payload: { accountID } });
+  try {
+    const filename = `report_for_${accountID}.tar.xz`;
+    const { data } = await Axios.get(
+      inventoryUrl(`${accountID}/uploads/file`),
+      {
+        responseType: 'blob',
+      }
+    );
+
+    const blob = new Blob([data], {
+      type: 'application/json',
+    });
+    // IE variant
+    if (window.navigator && window.navigator.msSaveOrOpenBlob) {
+      window.navigator.msSaveOrOpenBlob(blob, filename);
+    } else {
+      const url = window.URL.createObjectURL(blob);
+      const link = document.createElement('a');
+      link.href = url;
+      link.setAttribute('download', filename);
+      document.body.appendChild(link);
+      link.click();
```

Comment:
I wonder if you can use `import { saveAs } from 'file-saver';` here instead, foreman uses it in `TemplateGeneratorActions.js`